e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java

```diff
@@ -1671,13 +1680,39 @@ public SystemLogsAO setIncludeServiceAccountEventsOption() {
 
     public class MyProfileAO implements AccessObject<MyProfileAO> {
         private final Map<Primitive,SelenideElement> elements = initialiseElements(
-                entry(USER_NAME, $(byClassName("ser-profile__header")))
+                entry(USER_NAME, $(byClassName("ser-profile__header"))),
+                entry(LIMIT_MOUNTS, $(byClassName("limit-mounts-input__limit-mounts-input"))),
+                entry(DO_NOT_MOUNT_STORAGES, $(byXpath(".//span[.='Do not mount storages']/preceding-sibling::span")))
         );
 
         public MyProfileAO validateUserName(String user) {
             return ensure(USER_NAME, text(user));
         }
 
+        public SelectLimitMountsPopupAO<MyProfileAO> limitMountsPerUser() {
+            click(LIMIT_MOUNTS);
+            return new SelectLimitMountsPopupAO<>(this).sleep(2, SECONDS);
+        }
+
+        public MyProfileAO doNotMountStoragesSelect (boolean isSelected) {
```

Comment:
Remove a space between method name and parameter, please

Reply:
Done